parm/config/config.resources.emc.dyn

```diff
@@ -176,6 +176,14 @@ elif [ $step = "analdiag" ]; then
     if [[ "$npe_node_analdiag" -gt "$npe_node_max" ]]; then export npe_node_analdiag=$npe_node_max ; fi
     export memory_analdiag="48GB"
 
+elif [ $step = "wdqms" ]; then
+
+    export wtime_wdqms="00:10:00"
```

Comment:
The` gdaswdqms` in my global-workflow test requires more than 10 minutes.  So, I extended the wall time to 20 minutes.   Under testing...

Reply:
> The` gdaswdqms` in my global-workflow test requires more than 10 minutes. So, I extended the wall time to 20 minutes. Under testing...

Extending from 10 to 20 minutes, the `gdaswdqms` step competed in ~15 mintes.
Here are the resources used:
```
Job Id: 121655528.cbqs01
    Job_Name = v163_wdqms_gdaswdqms_12
    Job_Owner = emily.liu@clogin01.cactus.wcoss2.ncep.noaa.gov
    resources_used.cpupercent = 99
    resources_used.cput = 00:13:17
    resources_used.mem = 4011152kb
    resources_used.ncpus = 1
    resources_used.vmem = 1454824kb
    resources_used.walltime = 00:15:08

```

Reply:
DBN alert sent out for each CSV file.
```
915 + for otype in "${OTYPES[@]}"
915 + file=NCEP_SYNOP_20240311_12.csv
915 + cp ./NCEP_SYNOP_20240311_12.csv /lfs/h2/emc/ptmp/emily.liu/ROTDIRS/v163_wdqms/gdas.20240311/12/atmos/wdqms/NCEP_SYNOP_20240311_12.csv
915 + for otype in "${OTYPES[@]}"
915 + file=NCEP_TEMP_20240311_12.csv
915 + cp ./NCEP_TEMP_20240311_12.csv /lfs/h2/emc/ptmp/emily.liu/ROTDIRS/v163_wdqms/gdas.20240311/12/atmos/wdqms/NCEP_TEMP_20240311_12.csv
915 + for otype in "${OTYPES[@]}"
915 + file=NCEP_MARINE_20240311_12.csv
915 + cp ./NCEP_MARINE_20240311_12.csv /lfs/h2/emc/ptmp/emily.liu/ROTDIRS/v163_wdqms/gdas.20240311/12/atmos/wdqms/NCEP_MARINE_20240311_12.csv
915 + for otype in "${OTYPES[@]}"
915 + file=NCEP_SYNOP_20240311_12.csv
915 + [[ -f /lfs/h2/emc/ptmp/emily.liu/ROTDIRS/v163_wdqms/gdas.20240311/12/atmos/wdqms/NCEP_SYNOP_20240311_12.csv ]]
915 + echo 'Send DBN Alert'
Send DBN Alert
915 + for otype in "${OTYPES[@]}"
915 + file=NCEP_TEMP_20240311_12.csv
915 + [[ -f /lfs/h2/emc/ptmp/emily.liu/ROTDIRS/v163_wdqms/gdas.20240311/12/atmos/wdqms/NCEP_TEMP_20240311_12.csv ]]
915 + echo 'Send DBN Alert'
Send DBN Alert
915 + for otype in "${OTYPES[@]}"
915 + file=NCEP_MARINE_20240311_12.csv
915 + [[ -f /lfs/h2/emc/ptmp/emily.liu/ROTDIRS/v163_wdqms/gdas.20240311/12/atmos/wdqms/NCEP_MARINE_20240311_12.csv ]]
915 + echo 'Send DBN Alert'
Send DBN Alert
915 + echo 'Job completed normally.'
Job completed normally.
915 + exit 0
916 + echo 'JOB v163_wdqms_gdaswdqms_12 HAS COMPLETED NORMALLY!'

```